File: katas/8 kyu/remove-string-spaces/solution.py
import codewars_test as test
# -------------------------------- MY SOLUTION ------------------------------- #
def no_space(x):
    return "".join(x.split())
# ------------------------------ CLEVER SOLUTION ----------------------------- #
no_space = lambda x : x.replace(' ', '')
# ----------------------------------- TEST ----------------------------------- #
test.assert_equals(no_space('8 j 8   mBliB8g  imjB8B8  jl  B'), '8j8mBliB8gimjB8B8jlB')
test.assert_equals(no_space('8 8 Bi fk8h B 8 BB8B B B  B888 c hl8 BhB fd'), '88Bifk8hB8BB8BBBB888chl8BhBfd')
test.assert_equals(no_space('8aaaaa dddd r     '), '8aaaaaddddr')
test.assert_equals(no_space('jfBm  gk lf8hg  88lbe8 '), 'jfBmgklf8hg88lbe8') 
test.assert_equals(no_space('8j aam'), '8jaam')

File: katas/8 kyu/remove-string-spaces/solution.php
<!-- remove-string-spaces -->
<?php
use PHPUnit\Framework\TestCase;
/* ------------------------------- MY SOLUTION ------------------------------ */
function no_space(string $s): string {
  return str_replace(' ', '',$s);
}
/* ----------------------------- CLEVER SOLUTION ---------------------------- */
/* ---------------------------------- TEST ---------------------------------- */
class RemoveStringSpacesTest extends TestCase {
  public function testExamples() {
    $this->assertEquals('8j8mBliB8gimjB8B8jlB', no_space('8 j 8   mBliB8g  imjB8B8  jl  B'));
    $this->assertEquals('88Bifk8hB8BB8BBBB888chl8BhBfd', no_space('8 8 Bi fk8h B 8 BB8B B B  B888 c hl8 BhB fd'));
    $this->assertEquals('8aaaaaddddr', no_space('8aaaaa dddd r     '));
  }
}